location.js

```diff
@@ -0,0 +1,15 @@
+let nodeGeocoder = require('node-geocoder');
+ 
+let options = {
+  provider: 'openstreetmap'
+};
+ 
+let geoCoder = nodeGeocoder(options);
+
+geoCoder.geocode('8 nomad crescent, brampton')
+  .then((res)=> {
+    console.log(res[0]['latitude'],res[0]['longitude'] );
+  })
+  .catch((err)=> {
+    console.log(err);
+  });
```

Comment:
looks good!
Let's turn this into a service and put it in the server folder.
The idea is that when we're making a user we can take their address and store a geocode in their profile